cekit/tools.py

```diff
@@ -28,6 +28,8 @@ def get_cfg(config_path):
     cfg['common'] = cfg.get('common', {})
     cfg['common']['work_dir'] = cfg.get('common').get('work_dir', '~/.cekit')
     cfg['common']['redhat'] = cfg.get('common', {}).get('redhat', False)
+    if cp.has_option('common', 'addhelp'):
```

Comment:
maybe it would be nice to create `[doc]` section. You can have `addhelp` there and also probably a way to custom template.

Reply:
That sounds like a good idea. I'll implement the custom template stuff and then rework both to use [doc].